reactioncommerce/reaction
# Fix error in console when changing email in profile

Resolves #3306 

This PR fixes `TypeError: Cannot read property 'address' of undefined` error when changing email on the profile page.

## How to Test

- Login as admin
- Select "Profile" from dropdown
- Change email address
- Observe in the dev tool console that there is no error
